meetings/2017-03-08.md

```diff
@@ -0,0 +1,229 @@
+* _Previous Minutes Google Doc:
+  <https://docs.google.com/document/d/1xoSIVTwNoOfOf0NSajbZudD2HX4PvYQ5V9eVqfDu0qs>_
+
+## Present
+
+* Bradley Meck @bmeck (observer/GoDaddy/TC39)
+* Ben Noordhuis @bnoordhuis (CTC)
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+* Evan Lucas @evanlucas (CTC)
+* Franziska Hinkelmann @fhinkel (observer/Google)
+* Jeremiah Senkpiel @Fishrock123 (CTC)
+* James M Snell @jasnell (CTC)
+* Josh Gavant @joshgav (observer/Microsoft)
+* Michael Dawson @mhdawson (CTC)
+* Ali Ijaz Sheikh @ofrobots (CTC)
+* Jenn Turner @renrutnnej (observer/Node.js Foundation)
+
+
+## Standup
+
+* Bradley Meck @bmeck (observer/GoDaddy/TC39)
+  * Started porting loader impl to C++
+* Ben Noordhuis @bnoordhuis (CTC)
+  * same as always, nothing of note
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+  * Buffer-related things mostly
+* Evan Lucas @evanlucas (CTC)
+  * Cutting v7.7.2 release
+* Franziska Hinkelmann @fhinkel (observer/Google)
+  * Final week of Outreachy mentoring (vm module)
+* Jeremiah Senkpiel @Fishrock123 (CTC)
+  * Nothing particularly notable
+* James M Snell @jasnell (CTC)
+  * VM Summit
+  * Landing backports for v6 and v4
+* Josh Gavant @joshgav (observer/Microsoft)
+  * N-API, VM Summit
+  * diag stuff
+* Michael Dawson @mhdawson (CTC)
+  * VM Summit
+  * Ongoing work on N-API
+  * Build WG meeting
+  * Keeping up to date on issue, commenting, etc.
+* Brian White @mscdex (CTC)
+  * Continued work on optimizations in node core
+  * Commented on issues, reviewed PRs
+* Ali Ijaz Sheikh @ofrobots (CTC)
+  * VM Summit, participated in the presentation on FFI.
+* Jenn Turner @renrutnnej (observer/Node.js Foundation)
+  * No update, just observing
+
+
+## Agenda
+
+* governance: add Franziska Hinkelmann to the CTC
+  [node#11488](https://github.com/nodejs/node/pull/11488) @rvagg
+* Symbol key props visible in inspection by default
+  [node#9726](https://github.com/nodejs/node/pull/9726) @Trott
+* Updating the Copyright [TSC#174](https://github.com/nodejs/TSC/issues/174)
+  @jasnell
+* 002 - web compat of file extension/index searching
+  [node-eps#51](https://github.com/nodejs/node-eps/issues/51) @bmeck
+
+
+## Previous Meeting Review
+
+* tracking issue: async loader for module interop @trott
+  [node#11233](https://github.com/nodejs/node/issues/11233)
+
+WIP: https://github.com/nodejs/sketchy-as-heck-loader-impl
+
+Will open to all Collaborators.
+
+* Updating the Copyright @jasnell
+  [TSC#174](https://github.com/nodejs/TSC/issues/174)
+
+---
+
+## Minutes
+
+### governance: add Franziska Hinkelmann to the CTC [node#11488](https://github.com/nodejs/node/pull/11488)
+
+**VOTE (in progress)**
+* In favor (14): @addaleax, @bnoordhuis, @ChALkeR, @cjihrig, @evanlucas,
+  @Fishrock123, @jasnell, @mhdawson, @misterdjules, @MylesBorins, @ofrobots,
+  @targos, @thefourtheye, @Trott
+* Opposed (0):
+* Abstain (0):
+* Not yet voted (5): @indutny, @mscdex, @rvagg, @shigeki, @trevnorris
```

Comment:
Since @rvagg opened the pull request to add Franziska, it's probably safe to move him to "In favor".